internal/display/roles.go

```diff
@@ -58,6 +58,24 @@ func (r *Renderer) RoleList(roles []*management.Role) {
 	r.Results(res)
 }
 
+func (r *Renderer) UserRoleList(roles []*management.Role) {
+	resource := "roles assigned to user"
+	r.Heading(fmt.Sprintf("%s (%d)", resource, len(roles)))
+
+	if len(roles) == 0 {
+		r.EmptyState(resource)
+		r.Infof("Use 'auth0 users roles assign' to add one")
```

Comment:
```suggestion
		r.Infof("Use 'auth0 users roles assign' to assign roles to a user")
```

Reply:
Fixed in [`fb39943` (#604)](https://github.com/auth0/auth0-cli/pull/604/commits/fb399435d364e8cd1fa4bfbcfee284bc560dbdcc)